src/components/Buttons.js

```diff
@@ -23,3 +25,14 @@ export const OfflineButton = props => (
     Offline
   </OutButton>
 );
+
+export const CopyButton = ({ didCopy = false, ...props }) => (
+  <Button
+    accessory={
+      <WithTooltip content={didCopy ? 'Copied!' : 'Copy'}>
+        <Icon icon="Copy" size="18px" color={'black'} />
+      </WithTooltip>
+    }
+    {...props}
```

Comment:
Pretty sure we want `WithTooltip` around the entire button. Right now I can click the entire thing, but only see the feedback if I hover the icon.

Also, the tooltip feels a bit on the large side, font-wise. Might be nice to shrink it down to the size of the button's description.